source/_docs/guides/build-tools/07-custom-theme.md

```diff
@@ -0,0 +1,154 @@
+
+1. Start by creating a new branch from your local and pushing it up to GitHub (`custom-theme`):
+
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#git-branch">Copy</button>
+      <figure><pre id="git-branch"><code class="command bash" data-lang="bash">git checkout -b custom-theme
+      git push origin custom-theme</code></pre></figure>
+    </div>
+
+2. Wait for CircleCI to build a new Pantheon Multidev environment (`pr-custom-t`), then export environment variables to define your site name and Multidev environment to easily copy and paste example commands in the next sections (replace `pantheon-d8-composer-project`):
+
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#export-var1">Copy</button>
+      <figure><pre id="export-var1"><code class="command bash" data-lang="bash">export SITE=pantheon-d8-composer-project
+      export ENV=pr-custom-t</code></pre></figure>
+    </div>
+
+
+3. Use the `generate:theme` command as shown below to start the process of creating a subtheme:
+
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#drush-generate-theme">Copy</button>
+      <figure><pre id="drush-generate-theme"><code class="command bash" data-lang="bash">terminus drupal $SITE.$ENV -- generate:theme</code></pre></figure>
+    </div>
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+      <div class="panel-heading panel-drop-heading">
+         <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#understand-drupal-console"><h3 class="panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-lightbulb"></span> Drupal Console Generate Theme</h3></a>
+       </div>
+       <div id="understand-drupal-console" class="collapse">
+         <div class="panel-inner" markdown="1">
+        Drupal Console will ask a series of questions about your theme. For many of them you can use the default value by just hitting **Enter**. Provide additional information as prompted, such as:
+
+          * Theme: Amazing Theme
+          * Base theme: bartik
+          * Enter `no` to generate theme regions and theme breakpoints
+          * Type `yes` when asked to confirm generation of the theme
+
+        ![Drupal console generate theme](/source/docs/assets/images/pr-workflow/drupal-console-generate-theme.png)
+
+        Once you do this, the files for your new theme will be written to the directory `code/web/themes/custom/amazing_theme` on the Pantheon Multidev environment. If you gave your theme a different name, replace `amazing_theme` with the appropriate name for your theme.
+        </div>
+        </div>
+      </div>
+
+4. Open a second terminal window and run the following command to open an SFTP session from the command line:
```

Comment:
Maybe it would be better to have them commit their generated theme from the dashboard? We could advise adding "[ci skip]" in the commit message to [avoid running the tests on Circle](https://circleci.com/docs/1.0/skip-a-build/).

Reply:
Ooh, yeah I think doing a commit from the dashboard in this part would be smoother. I'll do a pass now to give it a shot. I'm confused on why we'd want to skip ci though, wouldn't we want the dashboard commit pushed to the GitHub repo @greg-1-anderson ?

Reply:
Just realizing that the push back to GitHub is a Quicksilver hook, not a build step on CirlceCI. 🌠 💥

Reply:
Just realizing that the push back to GitHub is a Quicksilver hook, not a build step on CirlceCI. 🌠 💥

Reply:
Correct, the pushback is done by Quicksilver. If you let the build step run, then it will overwrite the current multidev with the build results, presumably with the same code. This might also reset any configuration changes the user may have made without exporting.